sql/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -3759,6 +3759,18 @@ class SparkConnectPlanner(
         UnresolvedScalarSubqueryPlanId(planId)
       case proto.SubqueryExpression.SubqueryType.SUBQUERY_TYPE_EXISTS =>
         UnresolvedExistsPlanId(planId)
+      case proto.SubqueryExpression.SubqueryType.SUBQUERY_TYPE_TABLE_ARG =>
+        val options = getSubqueryExpression.getTableArgOptions
+        UnresolvedTableArgPlanId(
+          planId,
+          partitionSpec = options.getPartitionSpecList.asScala
+            .map(transformExpression)
+            .toSeq,
+          orderSpec = options.getOrderSpecList.asScala
+            .map(transformSortOrder)
+            .toSeq,
+          withSinglePartition = options.getWithSinglePartition
```

Comment:
Need to check if it has `with_single_partition`?

Reply:
Makes sense added.